File: app/src/main/java/com/tunasushi/bean/BindingBean.java
package com.tunasushi.bean;

import android.databinding.BaseObservable;
import android.databinding.ObservableBoolean;
import android.databinding.ObservableField;

/**
 * @author TunaSashimi
 * @date 2020-04-11 23:42
 * @Copyright 2020 TunaSashimi. All rights reserved.
 * @Description
 */
public class BindingBean extends BaseObservable {
    public ObservableField<String> name = new ObservableField("hello");
    public ObservableBoolean select = new ObservableBoolean();
}

File: app/src/main/java/com/tunasushi/bean/BindingBean.kt
package com.tunasushi.bean

import androidx.databinding.BaseObservable
import androidx.databinding.ObservableField
import androidx.databinding.ObservableBoolean

/**
 * @author TunaSashimi
 * @date 2020-04-11 23:42
 * @Copyright 2020 TunaSashimi. All rights reserved.
 * @Description
 */
class BindingBean : BaseObservable() {
    @JvmField
    var name: ObservableField<String?> = ObservableField("hello")
    @JvmField
    var select = ObservableBoolean()
}